go/cmd/vtctldclient/command/schema.go

```diff
@@ -138,7 +138,7 @@ func commandApplySchema(cmd *cobra.Command, args []string) error {
 
 	resp, err := client.ApplySchema(commandCtx, &vtctldatapb.ApplySchemaRequest{
 		Keyspace:                ks,
-		AllowLongUnavailability: applySchemaOptions.AllowLongUnavailability,
+		AllowLongUnavailability: false,
```

Comment:
this still exists in the `Use` [here](https://github.com/vitessio/vitess/blob/b44829f8afb7e22461ba5c0d3cb7ef2ce45644f9/go/cmd/vtctldclient/command/schema.go#L43).

Further, [here](https://github.com/vitessio/vitess/blob/b44829f8afb7e22461ba5c0d3cb7ef2ce45644f9/go/cmd/vtctldclient/command/schema.go#L289), you can add a line to deprecate the flag ([docs reference](https://pkg.go.dev/github.com/spf13/pflag#readme-deprecating-a-flag-or-its-shorthand)) via:

```go
ApplySchema.Flags().MarkDeprecated("--allow-long-unavailability", "<deprecation text goes here>")
```

Reply:
oooh! Didn't know that

Reply:
I've made the changes; but I'm left wondering why we have two different places where we define this flag? The other place being `vtctl/vtctl.go`

Reply:
One is the old vtctlclient command interface and the other is the new vtctldclient interface.

Reply:
This brings up another point. We should deprecate the proto field as well (using the `reserved` keyword) in vtctldata.proto.
@ajm188 do you agree?

Reply:
yes

Reply:
https://github.com/vitessio/vitess/pull/10716#discussion_r932594189

Reply:
In this case, the flag used to default to false, so deprecating the proto field will be fine (which is not the case in #10716)